source/options-storage.ts

```diff
@@ -41,7 +41,24 @@ export const renamedFeatures = new Map<string, string>([
 	['quick-pr-diff-options', 'one-click-diff-options'],
 	['quick-review-buttons', 'one-click-review-submission'],
 	['wait-for-build', 'wait-for-checks'],
```

Comment:
Let's change the format to this?

``` JavaScript
{
  "current-name": ["previous-names"]
} 
``` 

Advantages:

- no risk of duplicating the current feature because typescript will complain
- the type will be a stricter `Record<FeatureID, string[]>`